en,zh: add note for upgrade cmd after v1.3.0 (#1599)
pingcap/docs-tidb-operator: en/upgrade-tidb-operator.md

2. Update [CustomResourceDefinition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) (CRD) for Kubernetes:

    * If your Kubernetes version >= v1.16:
        

>         1. If upgrading Operator from v1.2.x or older to v1.3.x or later, you need to execute the following command to create the new TidbNGMonitoring CRD. If upgrading Operator from v1.3.x or later, you can skip this step.

Reviewer: ```suggestion
        1. If you upgrade TiDB Operator from v1.2.x or earlier versions to v1.3.x or later versions, you need to execute the following command to create the new TidbNGMonitoring CRD. If you upgrade TiDB Operator from v1.3.x or later versions, you can skip this step.
```

            {{< copyable "shell-regular" >}}

            ```shell
            kubectl create -f https://raw.githubusercontent.com/pingcap/tidb-operator/${operator_version}/manifests/crd/v1/pingcap.com_tidbngmonitorings.yaml
            ```
        
        2. Update CRD.

            {{< copyable "shell-regular" >}}

            ```bash
            kubectl replace -f https://raw.githubusercontent.com/pingcap/tidb-operator/${operator_version}/manifests/crd.yaml && \
            kubectl get crd tidbclusters.pingcap.com
            ```

    * If your Kubernetes version < v1.16:

>         1. If upgrading Operator from v1.2.x or older to v1.3.x or later, you need to execute the following command to create the new TidbNGMonitoring CRD. If upgrading Operator from v1.3.x or later, you can skip this step.

Reviewer: ```suggestion
        1. If you upgrade TiDB Operator from v1.2.x or earlier versions to v1.3.x or later versions, you need to execute the following command to create the new TidbNGMonitoring CRD. If you upgrade TiDB Operator from v1.3.x or later versions, you can skip this step.
```

    > **Note:**
    >

>     > If upgrading Operator to V1.3.0 or later for the first time, you can't use `replace` command to upgrade TidbNGMonitoring CRD due to TidbNGMonitoring CRD is new, you need to create TidbNGMonitoring first.

Author: TODO: 英文等中文 Review 后再修改

2. Upload the downloaded files and images to the server where TiDB Operator is deployed, and install the new TiDB Operator version:

>     1. If upgrading Operator from v1.2.x or older to v1.3.x or later, you need to exec the following command to create the new TidbNGMonitoring CRD. If upgrading Operator from v1.3.x or later, you can skip this step.

Reviewer: ```suggestion
    1. If you upgrade TiDB Operator from v1.2.x or earlier versions to v1.3.x or later versions, you need to execute the following command to create the new TidbNGMonitoring CRD. If you upgrade TiDB Operator from v1.3.x or later versions, you can skip this step.
```

        {{< copyable "shell-regular" >}}

        ```shell
        kubectl create -f ./crd.yaml
        ```

>         For other CRDs, there is a "AlreadyExists" error after executing this command, you can ingore this error.

Reviewer: ```suggestion
        After executing this command, you can expect to see an "AlreadyExists" error for other CRDs. You can ignore this error.
```
